tools/arxiv/scripts/run_weekend_test.sh

```
@@ -0,0 +1,187 @@
+TOTAL_PAPERS=$(wc -l < "$PAPER_LIST")
+echo ""
+echo -e "${GREEN}📄 Using paper list: $PAPER_LIST${NC}"
+echo -e "${GREEN}📊 Total papers available: $TOTAL_PAPERS${NC}"
+echo ""
+
+# Step 2: Clean up staging area
+echo -e "${YELLOW}Step 2: Cleanup${NC}"
+echo "Cleaning staging directory..."
+STAGING_DIR="/dev/shm/weekend_staging"
+if [ -d "$STAGING_DIR" ]; then
+    rm -rf "$STAGING_DIR"/*
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)